tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/DbScopeFactory.cs

```diff
@@ -40,18 +42,26 @@ private static Scope CreateDbCommandScope(Tracer tracer, IDbCommand command, Int
                     return null;
                 }
 
-                string serviceName = tracer.Settings.GetServiceName(tracer, dbType);
-
                 var tags = new SqlTags
                            {
                                DbType = dbType,
-                               InstrumentationName = integrationId.ToString()
+                               InstrumentationName = IntegrationRegistry.GetName(integrationId),
                            };
 
                 tags.SetAnalyticsSampleRate(integrationId, tracer.Settings, enabledWithGlobalSetting: false);
 
+                var commandTags = DbCommandCache.GetTagsFromDbCommand(command);
+                if (commandTags != null)
+                {
+                    var cachedTags = commandTags.Value;
```

Comment:
Maybe just me, but I prefer to have fewer "tags" variables around in scope 
```suggestion
                if (DbCommandCache.GetTagsFromDbCommand(command) is {} cachedTags)
                {
```

Reply:
I've removed the Nullable<T> in 769ed59